Angular/heroes-app/src/app/dashboard/dashboard.component.html

```diff
@@ -0,0 +1,9 @@
+<h3>Top Heroes</h3>
+
+<div>
+  <a *ngFor="let hero of heroes">
+      <h4 (click)="onSelect(hero)">{{hero.name}}</h4>
+  </a>
+</div>
+
+<app-hero-detail [hero]='selectedHero'></app-hero-detail>
```

Comment:
```suggestion
<app-hero-detail [hero]="selectedHero"></app-hero-detail>
```
In template we use double quotes